app/lib/server/methods/sendMessage.js

```diff
@@ -28,20 +27,7 @@ export function executeSendMessage(uid, message) {
 		});
 	}
 
-	if (message.ts) {
```

Comment:
Would be better to update the ts (on client side) before send it after became online? Otherwise the message can be inserted a long time in the post on the history, doesn't seems to be a good behavior.

Reply:
Fixed `ts` issue